index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module "react-easy-print" {
+	export const Print: any;
+	export const NoPrint: any;
+	const PrintProvider: any;
```

Comment:
I don't think this a proper way to do typing.
It'd be nice for you to actually put the types.

Reply:
+1, @zmilonas,
Can you add correct typings after the pr we’re preparing now: https://github.com/a-x-/react-easy-print/pull/19

Reply:
My reply seems outdated a bit,
I accepted looking correct typings here: https://github.com/a-x-/react-easy-print/pull/12
We’re try to update them after #19 pr will be merged